diff -ur a/conf/wacom.rules b/conf/wacom.rules
--- a/conf/wacom.rules	2014-05-28 22:16:32.000000000 +0300
+++ b/conf/wacom.rules	2015-01-02 22:42:00.660259955 +0200
@@ -11,7 +11,7 @@
 # comment out the next line if your distribution does not provide systemd
 # If a /dev/ttySx device with the WACf name is detected, start the
 # wacom-inputattach service with the kernel name as parameter
-SUBSYSTEM=="tty|pnp", KERNEL=="ttyS[0-9]*", ATTRS{id}=="WACf*", TAG+="systemd", ENV{SYSTEMD_WANTS}+="wacom-inputattach@%k.service"
+#SUBSYSTEM=="tty|pnp", KERNEL=="ttyS[0-9]*", ATTRS{id}=="WACf*", TAG+="systemd", ENV{SYSTEMD_WANTS}+="wacom-inputattach@%k.service"
 
 LABEL="wacom_end"
 
